troposphere/static/js/components/modals/instance/InstanceLaunchWizardModal.jsx

```diff
@@ -462,7 +462,7 @@ const InstanceLaunchWizardModal = React.createClass({
         });
     },
 
-    filterSizeList(sizes, imageVersion) {
+    filterSizeList(provider, sizes, imageVersion) {
```

Comment:
It's not in `master`, but perhaps I've neglected a merge forward, and I can check that and drop this out with an interactive rebase.